lib/graph/graph.js

```diff
@@ -54,15 +54,12 @@ export async function graphFromPackageDependencies({
 	cwd = cwd ? path.resolve(cwd) : process.cwd();
 	rootConfigPath = utils.resolveConfigPath(cwd, rootConfigPath);
 
-	let workspace;
-	if (workspaceName || workspaceConfiguration) {
```

Comment:
`workspaceName` can still be falsy, so I think the if-clause shouldn't be removed.

We should actually also allow `null` as a value. See my code suggestion above.

Reply:
Thanks! I have updated the tests to reflect that